homeassistant/components/sensor/azure_vm.py

```diff
@@ -0,0 +1,80 @@
+"""
+Sensor platform representing an Azure Virtual Machine.
+
+For more details about this sensor platform, please refer to the documentation
+at https://home-assistant.io/components/sensor.azure_vm/
```

Comment:
Please update after renaming the module.

Reply:
Please refer to my comment about multiple sensors based on azure